navigator/src/main/java/dev/ikm/komet/navigator/graph/treetasks/ExpandTask.java

```diff
@@ -96,39 +94,14 @@ protected Void compute() {
             pathParent.get().getChildren().addAll(childrenVertexes);
             pathParent.set(newPathParent.get());
         }
-        // Compute the row number, treeView.getRow(vertexToSelect) is to slow...
-
-
-        int currentRow = 0;
-        for (int i = 0; i < pathParentList.size() - 1; i++) {
-            MultiParentVertexImpl vertex = pathParentList.get(i);
-            MultiParentVertexImpl nextVertex = pathParentList.get(i + 1);
-
-            for (TreeItem childVertex : vertex.getChildren()) {
-                currentRow++;
-                if (childVertex == nextVertex) {
-                    break;
-                }
-            }
-        }
-
-        final int rowNumber = currentRow;
-
-        // Create an entire new root...
         Platform.runLater(() -> {
-            LOG.info("Sending expansion results to interface for: " + expansionPath +
-                    "\n Selecting row: " + rowNumber);
-            Platform.runLater(() -> {
-                multiParentGraphViewController.getTreeView().getRoot().getChildren().add(newTreeTop);
-                newTreeTop.setExpanded(true);
-                MultiParentVertexImpl vertexToSelect = pathParentList.get(pathParentList.size() - 1);
-                multiParentGraphViewController.getTreeView().scrollTo(rowNumber);
-                // +1 for the root concept, and +1 for the current?
-                // TODO: figure out why this is consistently off by 2, and update accordingly.
-                int rowToSelect = rowNumber + 2;
-                multiParentGraphViewController.getTreeView().getSelectionModel().select(rowToSelect);
-                multiParentGraphViewController.getTreeView().getSelectionModel().getSelectedItems().getFirst().setExpanded(true);
-            });
+            multiParentGraphViewController.getTreeView().getRoot().getChildren().add(newTreeTop);
```

Comment:
Can we add a comment that describes the code below?

Reply:
Added the code comments.